server/game/GameActions/PlaceFateAction.js

```diff
@@ -6,19 +6,27 @@ class PlaceFateAction extends CardGameAction {
     setDefaultProperties() {
         this.amount = 1;
         this.origin = null;
+        this.recipient = null;
     }
 
     setup() {
         this.name = 'placeFate';
-        this.targetType = [CardTypes.Character];
+        this.targetType = [CardTypes.Character, 'player'];
```

Comment:
I'm not particularly happy about this change because it means the game will let you place fate on a player, which shouldn't really be possible.  I think this change alone should fix all the issues though